project_test/lib/channels.py

```diff
@@ -0,0 +1,57 @@
+import time
+
+from core_data_modules.cleaners import Codes 
+from core_data_modules.traced_data import Metadata
+from dateutil.parser import isoparse
+
+class Channels(object):
+    #TODO: Setup rapid pro shows to test Ad,promos,shows & date ranges then update the list below.
+    NON_LOGICAL_KEY = "non_logical_time"
+    INTERNET_KEY = "internet_working"
+    WATER_KEY =  "water_filter"
+    WASTE_KEY = "waste_disposal"
+
+    # Time ranges expressed in format (start_of_range_inclusive, end_of_range_exclusive)
+    INTERNET_RANGES = [("2019-01-01T00:00:00+03:00","2019-04-01T00:00:00+03:00")]
+    WATER_RANGES = [("2019-01-01T00:00:00+03:00","2019-04-01T00:00:00+03:00")]
+    WASTE_RANGES = [("2019-01-01T00:00:00+03:00","2019-04-01T00:00:00+03:00")]
```

Comment:
Following my offline questions about flow configurations for this project, please add a comment explaining why all these time ranges are the same.

Reply:
Thanks @as2388.